tests/NgTests/TestableFeed2CatalogJob.cs

```diff
@@ -61,12 +61,12 @@ internal class TestLogger : ILogger
     {
         public void Log(LogLevel logLevel, int eventId, object state, Exception exception, Func<object, Exception, string> formatter)
         {
-            Console.WriteLine($"{logLevel}: {formatter(state, exception)}");
+            // Console.WriteLine($"{logLevel}: {formatter(state, exception)}");
```

Comment:
This was causing tests to have a lot of stdout (locally and on CI). 99% of the time, you only fail about the failure result, not the stdout.